isb_lib/smithsonian_adapter.py

```diff
@@ -93,13 +94,16 @@ def _validate_resolved_content(thing: Thing):
 def reparse_as_core_record(thing: Thing) -> typing.List[typing.Dict]:
     _validate_resolved_content(thing)
     try:
-        transformer = isamples_metadata.SmithsonianTransformer.SmithsonianTransformer(
-            thing.resolved_content
-        )
-        solr_doc = isb_lib.core.coreRecordAsSolrDoc(transformer)
-        # This isn't present in Smithsonian data.  Fall back to the value on Thing
-        solr_doc["sourceUpdatedTime"] = isb_lib.core.datetimeToSolrStr(thing.tstamp)
-        return [solr_doc]
```

Comment:
Another real bug -- if we ever had a degenerate Thing record with no resolved_content this would have blown up.  Now, you could argue the Smithsonian should never have degenerate things since we're getting them from a local dump, but…